### ISSUE TITLE
Error: "Could not create the Java Virtual Machine"

### ISSUE THREAD

**Issue by emhumphries** [Nesvilab/FragPipe#355]
Hi All,

I'm trying to perform a closed search for a TMT16 workflow however I've got the same error as https://github.com/Nesvilab/MSFragger/issues/19 except using more recent updates of FragPipe and Java. I've tried reinstalling Java and rebooting the computer with no success. Do you have any suggestions to help solve this issue? Thank you in advance, Erin Humphries.

`System OS: Windows Server 2016, Architecture: AMD64
Java Info: 1.8.0_281, Java HotSpot(TM) 64-Bit Server VM, Oracle Corporation

Version info:
FragPipe version 15.0
MSFragger version 3.2
Philosopher version 3.4.13 (build 1611589727)

LCMS files:
  Experiment/Group: 
  - C:\Users\ehumphries\Documents\MSFragger\210414_output\7\210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.mzML

18 commands to execute:
WorkspaceCleanInit [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe workspace --clean --nocheck
WorkspaceCleanInit [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe workspace --init --nocheck
MSFragger [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
java -jar -Dfile.encoding=UTF-8 -Xmx-1G C:\Progs\MSFragger-3.2\MSFragger-3.2\MSFragger-3.2.jar C:\Users\ehumphries\Documents\MSFragger\210414_output\fragger.params C:\Users\ehumphries\Documents\MSFragger\210414_output\7\210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.mzML
MSFragger move pepxml
java -cp C:\Progs\FragPipe-15.0\fragpipe\lib\fragpipe-15.0.jar;/C:/Progs/FragPipe-15.0/fragpipe/lib/commons-io-2.6.jar com.github.chhh.utils.FileMove C:\Users\ehumphries\Documents\MSFragger\210414_output\7\210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pepXML C:\Users\ehumphries\Documents\MSFragger\210414_output\210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pepXML
PeptideProphet: Workspace init [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output\fragpipe-210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pepXML-temp]
C:\Progs\philosopher_v3.4.13\philosopher.exe workspace --init --nocheck
PeptideProphet [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output\fragpipe-210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pepXML-temp]
C:\Progs\philosopher_v3.4.13\philosopher.exe peptideprophet --decoyprobs --ppm --accmass --nonparam --expectscore --decoy rev_ --database C:\Users\ehumphries\Documents\MSFragger\2021-04-14-decoys-contam-20190717_RRP_SWISS_Canon_human_with_BUGS.fasta.fas ..\210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pepXML
PeptideProphet: Delete temp
java -cp C:\Progs\FragPipe-15.0\fragpipe\lib\fragpipe-15.0.jar com.github.chhh.utils.FileDelete C:\Users\ehumphries\Documents\MSFragger\210414_output\fragpipe-210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pepXML-temp
Rewrite pepxml [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
java -cp C:\Progs\FragPipe-15.0\fragpipe\lib/* com.dmtavt.fragpipe.util.RewritePepxml C:\Users\ehumphries\Documents\MSFragger\210414_output\interact-210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pep.xml C:\Users\ehumphries\Documents\MSFragger\210414_output\7\210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.mzML
Delay
java -cp C:\Progs\FragPipe-15.0\fragpipe\lib/* com.dmtavt.fragpipe.util.DoNothing 1000
PtmProphet [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe ptmprophet --keepold --static --em 1 --nions b --mods STY:79.966331,M:15.9949 --minprob 0.5 --maxthreads 63 interact-210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pep.xml
ProteinProphet [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe proteinprophet --maxppmdiff 2000000 --output combined C:\Users\ehumphries\Documents\MSFragger\210414_output\interact-210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.pep.xml
PhilosopherDbAnnotate [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe database --annotate C:\Users\ehumphries\Documents\MSFragger\2021-04-14-decoys-contam-20190717_RRP_SWISS_Canon_human_with_BUGS.fasta.fas --prefix rev_
PhilosopherFilter [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe filter --sequential --razor --prot 0.01 --tag rev_ --pepxml C:\Users\ehumphries\Documents\MSFragger\210414_output --protxml C:\Users\ehumphries\Documents\MSFragger\210414_output\combined.prot.xml
FreeQuant [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe freequant --ptw 0.4 --tol 10 --isolated --dir C:\Users\ehumphries\Documents\MSFragger\210414_output\7
Quant (Isobaric) [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe labelquant --tol 20 --level 2 --plex 16 --annot C:\Users\ehumphries\Documents\MSFragger\210414_output\7\annotation.txt --brand tmt --dir C:\Users\ehumphries\Documents\MSFragger\210414_output\7
PhilosopherReport [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe report
TmtIntegrator [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
java -cp "C:\Progs\FragPipe-15.0\fragpipe\tools\tmt-integrator-2.4.0.jar" TMTIntegrator C:\Users\ehumphries\Documents\MSFragger\210414_output\tmt-integrator-conf.yml C:\Users\ehumphries\Documents\MSFragger\210414_output\psm.tsv
WorkspaceClean [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe workspace --clean --nocheck
~~~~~~~~~~~~~~~~~~~~~~

Execution order:

    Cmd: [START], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [WorkspaceCleanInit], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [MSFragger], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [PeptideProphet], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [PtmProphet], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [ProteinProphet], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [PhilosopherDbAnnotate], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [PhilosopherFilter], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [FreeQuant], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [Quant (Isobaric)], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [PhilosopherReport], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [TmtIntegrator], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]
    Cmd: [WorkspaceClean], Work dir: [C:\Users\ehumphries\Documents\MSFragger\210414_output]

~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~ fragpipe.config ~~~~~~~~~
# FragPipe v15.0ui state cache

AdjustFragIntensity=true
BoostComplementaryIon=false
CorrThreshold=0
DeltaApex=0.2
ExportPrecursorPeak=false
RFmax=500
RPmax=25
RTOverlap=0.3
SE.EstimateBG=false
SE.IsoPattern=0.3
SE.MS1PPM=10
SE.MS2PPM=20
SE.MS2SN=1.1
SE.MassDefectFilter=true
SE.MassDefectOffset=0.1
SE.NoMissedScan=1
SE.SN=1.1
crystalc.run-crystalc=false
database.db-path=C\:\\Users\\ehumphries\\Documents\\MSFragger\\2021-04-14-decoys-contam-20190717_RRP_SWISS_Canon_human_with_BUGS.fasta.fas
database.decoy-tag=rev_
fragpipe-config.bin-msfragger=C\:\\Progs\\MSFragger-3.2\\MSFragger-3.2\\MSFragger-3.2.jar
fragpipe-config.bin-philosopher=C\:\\Progs\\philosopher_v3.4.13\\philosopher.exe
fragpipe-config.bin-python=C\:\\ProgramData\\Anaconda3\\python
freequant.mz-tol=10
freequant.rt-tol=0.4
freequant.run-freequant=true
ionquant.excludemods=
ionquant.heavy=
ionquant.imtol=0.05
ionquant.ionfdr=0.01
ionquant.light=
ionquant.mbr=0
ionquant.mbrimtol=0.05
ionquant.mbrmincorr=0
ionquant.mbrrttol=1
ionquant.mbrtoprun=10
ionquant.medium=
ionquant.minexps=1
ionquant.minfreq=0.5
ionquant.minions=1
ionquant.minisotopes=1
ionquant.minscans=3
ionquant.mztol=10
ionquant.normalization=1
ionquant.peptidefdr=1
ionquant.proteinfdr=1
ionquant.proteinquant=1
ionquant.requantify=1
ionquant.rttol=0.4
ionquant.run-ionquant=false
ionquant.tp=3
ionquant.writeindex=0
msfragger.Y_type_masses=
msfragger.add_topN_complementary=0
msfragger.allowed_missed_cleavage=2
msfragger.calibrate_mass=2
msfragger.clip_nTerm_M=true
msfragger.data_type=0
msfragger.deisotope=1
msfragger.delta_mass_exclude_ranges=(-1.5,3.5)
msfragger.deneutralloss=1
msfragger.diagnostic_fragments=
msfragger.diagnostic_intensity_filter=0
msfragger.digest_max_length=50
msfragger.digest_min_length=7
msfragger.fragment_ion_series=b,y
msfragger.fragment_mass_tolerance=20
msfragger.fragment_mass_units=1
msfragger.intensity_transform=0
msfragger.ion_series_definitions=
msfragger.isotope_error=0/1/2/3
msfragger.labile_search_mode=off
msfragger.localize_delta_mass=false
msfragger.mass_diff_to_variable_mod=0
msfragger.mass_offsets=0
msfragger.max_fragment_charge=2
msfragger.max_variable_mods_combinations=5000
msfragger.max_variable_mods_per_peptide=3
msfragger.min_fragments_modelling=2
msfragger.min_matched_fragments=4
msfragger.minimum_peaks=15
msfragger.minimum_ratio=0.01
msfragger.misc.fragger.clear-mz-hi=0
msfragger.misc.fragger.clear-mz-lo=0
msfragger.misc.fragger.digest-mass-hi=5000
msfragger.misc.fragger.digest-mass-lo=500
msfragger.misc.fragger.enzyme-dropdown=trypsin
msfragger.misc.fragger.precursor-charge-hi=4
msfragger.misc.fragger.precursor-charge-lo=1
msfragger.misc.fragger.remove-precursor-range-hi=1.5
msfragger.misc.fragger.remove-precursor-range-lo=-1.5
msfragger.misc.slice-db=1
msfragger.num_enzyme_termini=2
msfragger.output_format=pepXML
msfragger.output_max_expect=50
msfragger.output_report_topN=1
msfragger.override_charge=false
msfragger.precursor_mass_lower=-50
msfragger.precursor_mass_mode=selected
msfragger.precursor_mass_units=1
msfragger.precursor_mass_upper=50
msfragger.precursor_true_tolerance=20
msfragger.precursor_true_units=1
msfragger.remove_precursor_peak=0
msfragger.report_alternative_proteins=false
msfragger.restrict_deltamass_to=all
msfragger.run-msfragger=true
msfragger.search_enzyme_butnotafter=P
msfragger.search_enzyme_cutafter=KR
msfragger.search_enzyme_name=trypsin
msfragger.table.fix-mods=0.000000,C-Term Peptide,true,-1; 0.000000,N-Term Peptide,true,-1; 0.000000,C-Term Protein,true,-1; 0.000000,N-Term Protein,true,-1; 0.000000,G (glycine),true,-1; 0.000000,A (alanine),true,-1; 0.000000,S (serine),true,-1; 0.000000,P (proline),true,-1; 0.000000,V (valine),true,-1; 0.000000,T (threonine),true,-1; 57.021464,C (cysteine),true,-1; 0.000000,L (leucine),true,-1; 0.000000,I (isoleucine),true,-1; 0.000000,N (asparagine),true,-1; 0.000000,D (aspartic acid),true,-1; 0.000000,Q (glutamine),true,-1; 0.000000,K (lysine),true,-1; 0.000000,E (glutamic acid),true,-1; 0.000000,M (methionine),true,-1; 0.000000,H (histidine),true,-1; 0.000000,F (phenylalanine),true,-1; 0.000000,R (arginine),true,-1; 0.000000,Y (tyrosine),true,-1; 0.000000,W (tryptophan),true,-1; 0.000000,B ,true,-1; 0.000000,J,true,-1; 0.000000,O,true,-1; 0.000000,U,true,-1; 0.000000,X,true,-1; 0.000000,Z,true,-1
msfragger.table.var-mods=15.994900,M,true,3; 42.010600,[^,true,1; 79.966330,STY,true,2; -17.026500,nQnC,false,1; -18.010600,nE,false,1; 4.025107,K,false,2; 6.020129,KR,false,2; 8.014199,K,false,2; 10.008269,R,false,2; 0.000000,site_10,false,1; 0.000000,site_11,false,1; 0.000000,site_12,false,1; 0.000000,site_13,false,1; 0.000000,site_14,false,1; 0.000000,site_15,false,1; 0.000000,site_16,false,1
msfragger.track_zero_topN=0
msfragger.use_topN_peaks=150
msfragger.write_calibrated_mgf=false
msfragger.zero_bin_accept_expect=0
msfragger.zero_bin_mult_expect=1
peptide-prophet.cmd-opts=--decoyprobs --ppm --accmass --nonparam --expectscore
peptide-prophet.combine-pepxml=false
peptide-prophet.run-peptide-prophet=true
phi-report.dont-use-prot-proph-file=false
phi-report.filter=--sequential --razor --prot 0.01
phi-report.pep-level-summary=false
phi-report.print-decoys=false
phi-report.run-report=true
protein-prophet.cmd-opts=--maxppmdiff 2000000
protein-prophet.run-protein-prophet=true
ptmprophet.cmdline=--keepold --static --em 1 --nions b --mods STY\:79.966331,M\:15.9949 --minprob 0.5
ptmprophet.run-ptmprophet=true
ptmshepherd.annotation-common=false
ptmshepherd.annotation-custom=false
ptmshepherd.annotation-glyco=false
ptmshepherd.annotation-unimod=true
ptmshepherd.annotation_file=
ptmshepherd.annotation_tol=0.01
ptmshepherd.cap_y_ions=
ptmshepherd.diag_ions=
ptmshepherd.glyco_mode=false
ptmshepherd.histo_smoothbins=2
ptmshepherd.iontype_a=false
ptmshepherd.iontype_b=true
ptmshepherd.iontype_c=false
ptmshepherd.iontype_x=false
ptmshepherd.iontype_y=true
ptmshepherd.iontype_z=false
ptmshepherd.localization_allowed_res=all
ptmshepherd.localization_background=4
ptmshepherd.normalization-psms=true
ptmshepherd.normalization-scans=false
ptmshepherd.output_extended=false
ptmshepherd.peakpicking_mass_units=0
ptmshepherd.peakpicking_minPsm=10
ptmshepherd.peakpicking_promRatio=0.3
ptmshepherd.peakpicking_width=0.002
ptmshepherd.precursor_mass_units=0
ptmshepherd.precursor_tol=0.01
ptmshepherd.remainder_masses=
ptmshepherd.run-shepherd=false
ptmshepherd.spectra_maxfragcharge=2
ptmshepherd.spectra_ppmtol=20
ptmshepherd.varmod_masses=Failed_Carbamidomethylation\:-57.021464
quantitation.run-label-free-quant=false
run-diaumpire=false
speclibgen.easypqp.extras.max_delta_ppm=15
speclibgen.easypqp.extras.max_delta_unimod=0.02
speclibgen.easypqp.extras.rt_lowess_fraction=0.01
speclibgen.easypqp.rt-cal=noiRT
speclibgen.easypqp.select-file.text=
speclibgen.run-speclibgen=false
speclibgen.use-easypqp=true
speclibgen.use-spectrast=false
tmtintegrator.add_Ref=1
tmtintegrator.aggregation_method=0
tmtintegrator.allow_overlabel=true
tmtintegrator.allow_unlabeled=true
tmtintegrator.best_psm=true
tmtintegrator.channel_num=16
tmtintegrator.dont-run-fq-lq=false
tmtintegrator.groupby=0
tmtintegrator.max_pep_prob_thres=0
tmtintegrator.min_ntt=0
tmtintegrator.min_pep_prob=0.9
tmtintegrator.min_percent=0.05
tmtintegrator.min_purity=0.5
tmtintegrator.min_site_prob=-1
tmtintegrator.mod_tag=none
tmtintegrator.ms1_int=true
tmtintegrator.outlier_removal=true
tmtintegrator.print_RefInt=false
tmtintegrator.prot_exclude=none
tmtintegrator.prot_norm=1
tmtintegrator.psm_norm=false
tmtintegrator.quant_level=2
tmtintegrator.ref_tag=Bridge
tmtintegrator.run-tmtintegrator=true
tmtintegrator.top3_pep=true
tmtintegrator.unique_gene=0
tmtintegrator.unique_pep=false
workdir=C\:\\Users\\ehumphries\\Documents\\MSFragger\\210414_output
workflow.input.data-type.im-ms=false
workflow.input.data-type.regular-ms=true
workflow.ram=0
workflow.threads=63

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
WorkspaceCleanInit [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe workspace --clean --nocheck
Process 'WorkspaceCleanInit' finished, exit code: 0
WorkspaceCleanInit [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
C:\Progs\philosopher_v3.4.13\philosopher.exe workspace --init --nocheck
INFO[10:10:06] Executing Workspace  v3.4.13                 
INFO[10:10:06] Removing workspace                           
INFO[10:10:06] Done                                         
Process 'WorkspaceCleanInit' finished, exit code: 0
INFO[10:10:06] Executing Workspace  v3.4.13                 
INFO[10:10:06] Creating workspace                           
INFO[10:10:06] Done                                         
MSFragger [Work dir: C:\Users\ehumphries\Documents\MSFragger\210414_output]
java -jar -Dfile.encoding=UTF-8 -Xmx-1G C:\Progs\MSFragger-3.2\MSFragger-3.2\MSFragger-3.2.jar C:\Users\ehumphries\Documents\MSFragger\210414_output\fragger.params C:\Users\ehumphries\Documents\MSFragger\210414_output\7\210408_EMH_T1-2_ADV_Phos_QE1_D_1_05.mzML
Invalid maximum heap size: -Xmx-1G
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
Process 'MSFragger' finished, exit code: 1
Process returned non-zero exit code, stopping
~~~~~~~~~~~~~~~~~~~~
Cancelling 16 remaining tasks
10:10:06 ERROR: Error writing log to file
java.nio.channels.ClosedByInterruptException: null
	at java.nio.channels.spi.AbstractInterruptibleChannel.end(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at java.nio.channels.Channels.writeFullyImpl(Unknown Source)
	at java.nio.channels.Channels.writeFully(Unknown Source)
	at java.nio.channels.Channels.access$000(Unknown Source)
	at java.nio.channels.Channels$1.write(Unknown Source)
	at sun.nio.cs.StreamEncoder.writeBytes(Unknown Source)
	at sun.nio.cs.StreamEncoder.implWrite(Unknown Source)
	at sun.nio.cs.StreamEncoder.write(Unknown Source)
	at java.io.OutputStreamWriter.write(Unknown Source)
	at java.io.BufferedWriter.flushBuffer(Unknown Source)
	at java.io.BufferedWriter.close(Unknown Source)
	at com.dmtavt.fragpipe.tabs.TabRun.saveLogToFile(TabRun.java:355)
	at com.dmtavt.fragpipe.tabs.TabRun.on(TabRun.java:309)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:510)
	at org.greenrobot.eventbus.EventBus.postToSubscription(EventBus.java:447)
	at org.greenrobot.eventbus.EventBus.postSingleEventForEventType(EventBus.java:414)
	at org.greenrobot.eventbus.EventBus.postSingleEvent(EventBus.java:387)
	at org.greenrobot.eventbus.EventBus.post(EventBus.java:268)
	at com.dmtavt.fragpipe.api.Bus.post(Bus.java:51)
	at com.dmtavt.fragpipe.cmd.ProcessBuilderInfo.lambda$toRunnable$0(ProcessBuilderInfo.java:96)
	at java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
`

**Comment by fcyu** [comment]
> Invalid maximum heap size: -Xmx-1G

Can you tell us how much free memory available in your computer?

Best,

Fengchao

**Comment by emhumphries** [comment]
Hi Fengchao,
Thanks for your help. I'm running on a server so I'm not sure which one below:
- Installed Physical Memory (RAM) 63.9 GB
- Total Physical Memory 3.88 GB
- Available Physical Memory 1.53 GB
- Total Virtual Memory 13.4 GB
- Available Virtual Memory 10.7 GB
Cheers, Erin Humphries

**Comment by fcyu** [comment]
I am also don't know Windows server very well, but "Available Physical Memory 1.53 GB" looks like the one we are looking for.

FragPipe automatically uses `(int) freeMem - 2` if users don't specified the memory explicitly. If you think your task can be done with 1 GB memory, you can put 1 in the `RAM` in the `workflow` tab:
![image](https://user-images.githubusercontent.com/6926299/114954930-907eaa00-9e29-11eb-8f25-07cf47938d3f.png). Otherwise, you need to free some memory in your computer.

We will put a warning in FragPipe if `(int) freeMem - 2 < 0` in the next release.

Best,

Fengchao

**Comment by emhumphries** [comment]
Manually setting the memory to 40GB solved the problem. Could be an MSFragger and windows server communication issue.
![image](https://user-images.githubusercontent.com/74599273/114955566-513f7000-9ea0-11eb-9d03-3ac3c0566509.png)
